sdk/core/azure-core/inc/azure/core/strings.hpp

```diff
@@ -2,25 +2,16 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @file
- * @brief Common definitions.
+ * @brief Public functions for strings.
+ *
  */
 
-#pragma once
-
-#include "azure/core/internal/contract.hpp"
-
 #include <string>
 
-/**
- * @brief Used in implementations to mark an unreferenced function parameter.
- */
-#define AZURE_UNREFERENCED_PARAMETER(x) ((void)(x));
```

Comment:
Why are we removing it?

Reply:
there is only one use of it and I am replacing it to use `(void)parameter` instead as all other places.